spec/factories/users.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :integer          not null, primary key
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  remember_created_at    :datetime
+#  sign_in_count          :integer          default(0), not null
+#  current_sign_in_at     :datetime
+#  last_sign_in_at        :datetime
+#  current_sign_in_ip     :inet
+#  last_sign_in_ip        :inet
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  verified               :boolean          default(FALSE), not null
+#  type                   :string
+#  first_name             :string           default("")
+#  last_name              :string           default("")
+#  authentication_token   :string
+#
+
+FactoryGirl.define do
+  factory :user do
+    
+  end
+
+end
```

Comment:
nit: There's trailing whitespace in this file. I'm guessing you haven't enabled hound. Can you add that (and Rubocop + Guard) as an issue for a future PR?